File: pgm/ABC_Java/ABC_326_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] arr = reader.readLine().split (" ");
        final String Yes = "Yes";
        final String No = "No";
        final int X, Y;
        X = Integer.parseInt (arr[0]);
        Y = Integer.parseInt (arr[1]);
        System.out.println (X < Y && Y - X <= 2    ? Yes
                            : X > Y && Y - X >= -3 ? Yes
                                                   : No);
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_326_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.*
// import java.util.StringTokenizer

// import kotlin.math.*

const val Yes: String = "Yes"
const val No: String = "No"
const val space: String = " "
const val two: Int = 2
const val three: Int = 3

fun main() {
    val (X: Int, Y: Int) = readln().split(space).map { it.toInt() }
    if (X < Y && Y - X <= two) {
        println(Yes)
    } else if (X > Y && X - Y <= three) {
        println(Yes)
    } else {
        println(No)
    }
    kotlin.system.exitProcess(0)
}